client/components/data/query-rewind-backup-status/index.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+import { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { delay } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getRewindBackupProgress } from 'state/activity-log/actions';
+
+class QueryRewindBackupStatus extends PureComponent {
+	static propTypes = {
+		freshness: PropTypes.number,
+		queryDelay: PropTypes.number.isRequired,
+		downloadId: PropTypes.number,
+		siteId: PropTypes.number.isRequired,
+	};
+
+	static defaultProps = {
+		queryDelay: 0,
+	};
+
+	query( props ) {
+		const { getBackupProgress, queryDelay, downloadId, siteId } = props;
+		if ( ( siteId, downloadId ) ) {
+			delay( getBackupProgress, queryDelay, siteId, downloadId );
```

Comment:
are we wanting to _delay_ this or _throttle_ it?

what's the intended behavior?

Reply:
That the query is done a few milliseconds later. Not the best approach IMHO. This should later be refactored and maybe try to do more stuff, like continuous polling, in data layer. I'll probably need your help there.

Reply:
what is the purpose of the delay in the first place?